### ISSUE TITLE
FEATURE REQUEST: underlining / waved underlining

### ISSUE THREAD

**Issue by eirinn1975** [OptimiliaStudios/AquileReader#56]
Just an idea: add underlining and waved underlining with different colours in addition to highlighting, it's just to have different choices to point at contents

**Comment by isildur13** [comment]
> Just an idea: add underlining and waved underlining with different colours in addition to highlighting, it's just to have different choices to point at contents

Yes !!  the app is great but this feature of underlining will be very cool..

**Comment by OptimiliaStudios** [comment]
Will look into how to incorporate this feature in the app
